### DISCUSSION TITLE
Question about Audio Embedding

### DISCUSSION THREAD

**Discussion by bayaireus** [maximevaillancourt/digital-garden-jekyll-template#128] [Q&A]
Is it possible to embed audio files (MP3s, WAV, etc. etc.) into our Digital Gardens w/ Netifly? I seem to be having troubles using HTML to embed audio files that are in my repository.

Edit: I solved the issue. The links weren't matching up.

**Comment by maximevaillancourt** [comment]
Glad to read that you solved the issue.

In case others find this discussion, here's an example I just added to the live template demo: https://github.com/maximevaillancourt/digital-garden-jekyll-template/pull/129
